src/main/java/io/anserini/analysis/TokenizeAnalyzer.java

```diff
@@ -0,0 +1,13 @@
+package io.anserini.analysis;
+
+import org.apache.lucene.analysis.Analyzer;
+import org.apache.lucene.analysis.standard.StandardTokenizer;
+
+public class TokenizeAnalyzer extends Analyzer {
```

Comment:
Would `TokenizeOnlyAnalyzer` be a better name?

Reply:
Yes I agree, will update it.